### DISCUSSION TITLE
Installing Vaultwarden on Turnkey Core Linux

### DISCUSSION THREAD

**Discussion by Shootify** [tteck/Proxmox#1627] [Help Needed] [answered]
Hi there guys, wondering if is possible to use these scripts on a Turnkey Core LInux, the advantage is turnkey Core comes wtth preinstalled easy management, so it wil be ideal, any feedback will be appreciated it. 
thanks

**Comment by tteck** [comment]
How about setting up a new Vaultwarden LXC and including Webmin System Administration, both of which are available in the repository. Wouldn't that achieve the same outcome?

**Comment by Shootify** [comment]
Thanks for the suggestion, the reason i am trying to use turnkey Core is cuz the easy management, Networking, Get LE SSL and much more... what will be the changes of the script to run the installation inside the container? thanks a lot for your replies.

**Reply by tteck** [reply, marked as answer]
Turnkey Core's "easy management" is Webmin.

> wondering if is possible to use these scripts on a Turnkey Core LInux

The script won't function as you're envisioning. So, no.

**Comment by Shootify** [comment]
alright, understood.
what will be the recommended way to obtain LE SSL inside the instance itself. Lets say i have a dedicated proxmox server with public ip stack and just want to deploy Vaultwarden using your script, what will be the best way to obtain LE SSL? thanks for the help.

**Reply by tteck** [reply]
I recommend using Nginx Proxy Manager, automatic LE SSL renewal.

FYI, here is a look at Webmin System Administration:

![Screenshot 2023-07-18 9 45 56 PM](https://github.com/tteck/Proxmox/assets/21342670/47315b5e-960e-4a69-a9da-2bc7593c0a13)

**Comment by Shootify** [comment]
Sorry if i am not mistaken, Nginx proxy manager is for on premise servers, mine is dedicated on the "cloud" with ip stack

**Reply by tteck** [reply]
I'm not familiar with the _"cloud" using the IP stack_ method. This repository caters more to the self-hosting community.

**Reply by Shootify** [reply]
this is pretty much the same, exept that my server is on a datacenter with proxmox installed and ip stack assigned.
